lambdas/retrieve-cri-credential/src/main/java/uk/gov/di/ipv/core/retrievecricredential/RetrieveCriCredentialHandler.java

```diff
@@ -199,7 +202,14 @@ public Map<String, Object> handleRequest(Map<String, String> input, Context cont
                 | CiPostMitigationsException e) {
             updateVisitedCredentials(
                     ipvSessionItem, credentialIssuerId, null, false, OAuth2Error.SERVER_ERROR_CODE);
+            if (credentialIssuerId.equals(DCMAW_CRI)
+                    && e instanceof VerifiableCredentialException vce
+                    && vce.getHttpStatusCode() == HTTPResponse.SC_NOT_FOUND) {
+                LOGGER.info("404 received from DCMAW CRI");
```

Comment:
Should we use `logHelper.logErrorMessage` for this? Something like [this](https://github.com/alphagov/di-ipv-core-back/pull/1111/files#diff-3a6dc091cd6afdce9e5ed4642061c3b13be88243c1b80b60c6cb85deec5a4fedR140-R142)

Reply:
Good shout I've updated it